src/System.Private.CoreLib/shared/System/Globalization/CompareInfo.Unix.cs

```diff
@@ -459,7 +460,7 @@ private unsafe int IndexOfOrdinalHelper(ReadOnlySpan<char> source, ReadOnlySpan<
             }
         }
 
-        private unsafe int LastIndexOfCore(string source, string target, int startIndex, int count, CompareOptions options)
+        private unsafe int LastIndexOfCore(string? source, string target, int startIndex, int count, CompareOptions options)
```

Comment:
Based on the Debug.Assert in 467, source shouldn't be nullable. (I know we should add string.IsNullOrEmpty support to the tool) 😄